kubernetes/config/kube_config.py

```diff
@@ -62,11 +62,12 @@ class FileOrData(object):
      raw data and data field is base64 string."""
```

Comment:
NIT: I may be wrong but shouldn't we update comment to reflect that raw data can be base64 encoded now?